main.js

```diff
@@ -15,18 +15,53 @@
     $(function() {
         var body = document.getElementsByTagName('body')[0];
 
+		// Inject breadcrumbs
```

Comment:
I'm concerned about global changes here ... can't this be moved to your page only?